README.md

```diff
@@ -113,15 +113,28 @@ If you are an Enterprise user with first-party hosting access, you will need to
 You can then enable it in your code:
 
 ``` swift
-public enum Endpoint {
-    case default, alternate
-}
+let hcaptcha = try? HCaptcha(
+    ...
+    endpointApi: URL("https://custom.api.endpoint")!,
+    endpoint: URL("https://custom.endpoint")!,
```

Comment:
Same comment as above.

Reply:
I have removed `endpointApi`

Also according the text above it's required for enterprise clients

```
If you are an Enterprise user with first-party hosting access, you will need to set your own endpoint.

You can then enable it in your code:
```

Is that true?

Reply:
It is only true *if that particular feature is used* (which is optional).